### PR TITLE
multi: change account session type and reserve type 4

### PR DESCRIPTION
**Pull request by ellemouton**
In this commit, the Account session type is changed to be type 5 in both the session proto and in the persisted Session. This is done so that type 4 can be reserved for future use.

### CODE REVIEW

File: litrpc/lit-sessions.proto [hunk 1]
```diff
@@ -15,11 +15,12 @@ service Sessions {
 }
 
 enum SessionType {
+    reserved 4;
```
Comment (guggero) [inline litrpc/lit-sessions.proto:18]: nit: keep this in the original order? So between 3 and 5? Or does the compiler complain then?

File: litrpc/lit-sessions.proto [hunk 2]
```diff
@@ -15,11 +15,12 @@ service Sessions {
...
     TYPE_MACAROON_READONLY = 0;
     TYPE_MACAROON_ADMIN = 1;
     TYPE_MACAROON_CUSTOM = 2;
     TYPE_UI_PASSWORD = 3;
-    TYPE_MACAROON_ACCOUNT = 4;
+    TYPE_MACAROON_ACCOUNT = 5;
```
Comment (guggero) [inline litrpc/lit-sessions.proto:23]: I think this might also require a re-build of the JS stuff?

File: litrpc/lit-sessions.proto (hunk 1 above)
Comment (ellemouton) [reply litrpc/lit-sessions.proto]: done 👍

File: litrpc/lit-sessions.proto (hunk 2 above)
Comment (ellemouton) [reply litrpc/lit-sessions.proto:23]: aaahhh good catch!! updated!

**Review by guggero — APPROVED** [review]
LGTM :tada:

**Review by bitromortac — APPROVED** [review]

**Pull request merged**